### DISCUSSION TITLE
Can't remove domain from Vercel

### DISCUSSION THREAD

**Discussion by hornta** [vercel/community#900] [Help] [answered]
When trying to remove a domain it says:

```
example.com cannot be removed since it's still using Vercel Nameservers. Please remove Vercel Nameservers to remove example.com.
```
_example.com_ is not the actual domain used.

The error tells me to `Please remove Vercel Nameservers to remove example.com.` but I have no idea where to do that. Help please!

**Comment by amyegan** [comment, marked as answer]
Hi @hornta! You will need to change the nameservers with your domain name registrar. For example, if you bought your domain through Google Domains then you would follow the steps outlined here: https://support.google.com/domains/answer/3290309. Or if you bought your domain through Namecheap, you could follow the steps outlined here: https://www.namecheap.com/support/knowledgebase/article.aspx/767/10/how-to-change-dns-for-a-domain/.

Whichever registrar you used should have a guide available to walk you through the process 🙂

**Reply by hornta** [reply]
@amyegan Our domain is not using vercel nameservers. We've setup everything at Cloudflare but we want this custom domain removed from Vercel. I believe something in the Vercel system is bugged or has ended up in a corrupted state not allowing us to delete the domain.

**Reply by amyegan** [reply]
Oh no! Just to be sure I understand: when you set up everything on Cloudflare you used `A` and `CNAME` records rather than [custom nameservers](https://developers.cloudflare.com/dns/additional-options/custom-nameservers/), correct?

If you are not using Vercel nameservers and still seeing this error, then you should open a new case at [vercel.com/help](https://vercel.com/help#issues). That will allow you and the team to privately discuss the details of your domain and account to get this problem sorted.
